proto/greptime/v1/region/server.proto

```diff
@@ -136,11 +137,19 @@ message AddColumns { repeated AddColumn add_columns = 1; }
 
 message DropColumns { repeated DropColumn drop_columns = 1; }
 
+message ChangeColumnTypes { repeated ChangeColumnType change_column_types = 1; }
+
 message AddColumn {
   RegionColumnDef column_def = 1;
   AddColumnLocation location = 3;
 }
 
+message ChangeColumnType {
```

Comment:
Could we reuse `ChangeColumnTypes` and `ChangeColumnType` in `ddl.proto`? They have the same definitions.

Reply:
I thought so at first, but I did it with reference to `DropColumn`, so I’m not sure either.

Reply:
cc @MichaelScofield @v0y4g3r @waynexia Shall we reuse messages defined in `ddl.proto`? Then we might also remove the `DropColumn` message in this file. This should be backward compatible.

Reply:
Makes sense to me. We can change others in a follow-up PR. And reduce the duplicate `ChangeColumnTypes` introduced in this PR.

Reply:
Tried removing `ChangeColumnTypes` and `ChangeColumnType` on `server.rs`, which resulted in a lot of code like `TryFrom<greptime_proto::v1::region::alter_request::Kind` in https://github.com/GreptimeTeam/greptimedb/pull/3796 It's difficult to be consistent, I don't think it's a good idea to solve this problem in this pr (it would be better to solve it together later)

Reply:
server.proto has imported ddl.proto, can we reuse the `message ChangeColumnTypes` defined there?

Reply:
> Tried removing `ChangeColumnTypes` and `ChangeColumnType` on `server.rs`, which resulted in a lot of code like `TryFrom<greptime_proto::v1::region::alter_request::Kind` in [GreptimeTeam/greptimedb#3796](https://github.com/GreptimeTeam/greptimedb/pull/3796)


I don't quite understand why we need many `TryFrom` as `ChangeColumnTypes` and `ChangeColumnType` are types newly added. The difference should be from where we import these types. In contrast, reusing the messages should avoid some conversions as we use the same type now.